src/config.ts

```diff
@@ -37,14 +37,65 @@ const clientConfig: ClientProps = {
     true
   ),
   enableLogging: envValueToBoolean(process.env.REACT_APP_OIDC_LOGGING, false),
-  tokenExchangePath: process.env.REACT_APP_OIDC_TOKEN_EXCHANGE_PATH
+  tokenExchangePath: process.env.REACT_APP_OIDC_TOKEN_EXCHANGE_PATH,
+  path: '/helsinkimvp',
+  routeMatch: '/helsinkimvp/:subpage?',
+  hasApiTokenSupport: true,
+  label: 'Helsinki-profiili MVP'
 };
 
 const uiConfig: { profileUIUrl: string } = {
   profileUIUrl: String(process.env.REACT_APP_PROFILE_UI_URL)
 };
 
+const plainSuomiFiConfig: ClientConfig = {
```

Comment:
I'm imagining that there could be a "configuration builder" function that could be used something like:

```javascript
const plainSuomifiConfig: ClientConfig = buildConfig('REACT_APP_PLAIN_SUOMIFI');
```

Reply:
Added. I left it out earlier, because returning back to single config would have been simpler.